Tenable sc enhancment (#26319)
demisto/content: Packs/Tenable_sc/Integrations/Tenable_sc/Tenable_sc_description.md

Use the Tenable.sc integration to get a real-time, continuous assessment of your security posture so you can find and fix vulnerabilities faster.

> All data in Tenable.sc is managed using group level permissions. If you have several groups, data (scans, scan results, assets, etc) can be viewable but not manageable. Users with Security Manager role  can manage everything. These permissions come into play when multiple groups are in use.

Reviewer: ```suggestion
All data in Tenable.sc is managed using group level permissions. If you have several groups, data (scans, scan results, assets, etc.) can be viewable but not manageable. Users with Security Manager role  can manage everything. These permissions come into play when multiple groups are in use.
```

It is important to know what data is manageable for the user in order to work with the integration.

## Use cases:

    * Create and run scans.
    * Launch and manage scan results and the found vulnerabilities.
    * Create and view assets.
    * View policies, repositories, credentials, users and more system information.

>     * View and real-time receiving of alerts.

Reviewer: ```suggestion
    * View alerts received in real-time.
```

**Added support for secret & access keys authentication method.** 

- Added support for secret & access keys authentication method (API Key Authentication) which can be generated from the Tenable SC UI while logged into the desired account.

> - secret & access keys needs to be generated twice, once for sec man and once for admin.

Reviewer: ```suggestion
- Secret & access keys needs to be generated twice, once for secman and once for admin.
```

- First, you need to enable API Key Authentication:

## [Steps to follow:](https://docs.tenable.com/security-center/Content/EnableAPIKeys.htm)

      1. Log in to Tenable Security Center via the user interface.

>       2. Go to System > Configuration.

Reviewer: ```suggestion
      2. Go to **System** > **Configuration**.
```

>       3. Click the Security tile.

Reviewer: ```suggestion
      3. Click the **Security** tile.
```

>       4. In the Authentication Settings section, click Allow API Keys to enable the toggle.

Reviewer: ```suggestion
      4. In the Authentication Settings section, click **Allow API Keys** to enable the toggle.
```

>       5. Click Submit.

Reviewer: ```suggestion
      5. Click **Submit**.
```

> - Second, you need to generate API Key: 

Reviewer: ```suggestion
- Second, you need to generate the API Key: 
```

## [Steps to follow:](https://docs.tenable.com/security-center/Content/GenerateAPIKey.htm)

      1. Log in to Tenable Security Center via the user interface.

>       2. Go to Users > Users.

Reviewer: ```suggestion
      2. Go to **Users** > **Users**.
```

      3. Right-click the row for the user for which you want to generate an API key.
        -or-

>         Select the check box for the user for which you want to generate an API key.

Reviewer: ```suggestion
        Select the checkbox for the user for which you want to generate an API key.
```

>       4. Click API Keys > Generate API Key.

Reviewer: ```suggestion
      4. Click **API Keys** > **Generate API Key**.
```

>       5. Click Generate.

Reviewer: ```suggestion
      5. Click **Generate**.
```
